src/voterData/voterOutreach.service.ts

```diff
@@ -0,0 +1,141 @@
+    private readonly filesService: FilesService,
+    private readonly campaignsService: CampaignsService,
+  ) {}
+
+  async scheduleOutreachCampaign(
+    user: User,
+    campaign: Campaign,
+    {
+      budget,
+      audience,
+      script,
+      date,
+      message,
+      voicemail,
+      type,
+    }: ScheduleOutreachCampaignSchema,
+    imageUpload: FileUpload,
+  ) {
+    const { firstName, lastName, email, phone } = user
+
+    // TODO: reimplement
+    // const crmCompany = await sails.helpers.crm.getCompany(campaign)
+    // if (!crmCompany) {
+    //   throw new Error(`crmCompany not found for ${campaign}`)
+    // }
+    // const assignedPa = await getCrmCompanyOwnerName(crmCompany, true)
+
+    const messagingScript: string = campaign.aiContent?.[script]?.content
+      ? sanitizeHtml(campaign.aiContent?.[script]?.content, {
+          allowedTags: [],
+          allowedAttributes: {},
+        })
+      : script
+
+    // build Voter File URL
+    let voterFileUrl
+    try {
+      const filters: string[] = []
+      for (const key in audience) {
+        if (audience[key] === true) {
+          filters.push(key)
+        }
+      }
+      const encodedFilters = encodeURIComponent(JSON.stringify({ filters }))
+      voterFileUrl = `${WEBAPP_ROOT}/${VOTER_FILE_ROUTE}?type=${type}&slug=${campaign.slug}&customFilters=${encodedFilters}`
+    } catch (e) {
+      this.logger.error('Error building voterFileUrl: ', e)
+      voterFileUrl = null
+    }
+
+    // format audience filters for slack message
+    const formattedAudience = Object.entries(audience)
+      .map(([key, value]) => {
+        if (key === 'audience_request') {
+          return
+        }
+
+        return {
+          type: 'rich_text_section',
+          elements: [
+            {
+              type: 'text',
+              text: `${key}: `,
+              style: {
+                bold: true,
+              },
+            },
+            {
+              type: 'text',
+              text: value ? '✅ Yes' : '❌ No',
+            },
+          ],
+        }
+      })
+      // eslint-disable-next-line eqeqeq
+      .filter((val) => val != undefined)
+
+    // Upload image
+    const bucket = `scheduled-campaign/${campaign.slug}/${type}/${date}`
+    const imageUrl = await this.filesService.uploadFile(imageUpload, bucket)
+
+    const slackBlocks = buildSlackBlocks({
+      name: `${firstName} ${lastName}`,
+      email,
+      phone, // TODO: reimplement assignedPa + crmCompanyId!!!
+      assignedPa: undefined, //assignedPa,
+      crmCompanyId: undefined, // crmCompany?.id,
+      voterFileUrl,
+      type,
+      budget,
+      voicemail,
+      date,
+      script,
+      messagingScript,
+      imageUrl,
+      message,
+      formattedAudience,
+      audienceRequest: audience['audience_request'],
+    })
+
+    await this.slack.message(
+      slackBlocks,
+      IS_PROD ? SlackChannel.botPolitics : SlackChannel.botDev,
+    )
```

Comment:
Don't _think_ we need to `await` this here.. we can just fire and forget most likely..